### ISSUE TITLE
Highlight any accessibility regressions from prior report

### ISSUE THREAD

**Issue by mgifford** [HTTPArchive/almanac.httparchive.org#4031]
We can do a better job in the report highlighting where we might have lost ground on accessibility.

**Comment by tunetheweb** [comment]
@mgifford maybe we could collect all a11y suggestions into a single issue? That way we can easily find them all inn one place and also keep the issues list a bit cleaner. WDYT?

Also the `accessibility` label is for a11y issues with the Web Almanac itself so going to remove this label if you don't mind.

**Comment by mgifford** [comment]
That could work. Mostly i don't want to lose them in slack.

**Comment by tunetheweb** [comment]
> That could work. Mostly i don't want to lose them in slack.

Cool. Can I leave you to consolidate them all in one issue and close out the others?
